### PR TITLE
Update evaluation samples to use azure-ai-evaluation over promptflow-evals

### PR DESCRIPTION
**Pull request by slister1001**
# Description

Updates the samples with dependencies on promptflow-evals to depend on azure-ai-evaluation 

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure-Samples/azureai-samples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Review by nagkumar91 — APPROVED** [review]

**Comment by kdestin** [comment]
Thank you for your PR.

Please take a look at our contributing guidelines for guidance on how to set up `pre-commit` locally, so you can resolve the CI failures: https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md#3-set-up-pre-commit

**Review by ninghu — APPROVED** [review]

File: scenarios/evaluate-app-endpoint/data.jsonl [hunk 1]
```diff
@@ -1,3 +1,3 @@
-{"question":"When was United Stated found ?", "ground_truth":"1776"}
-{"question":"What is the capital of France?", "ground_truth":"Paris"}
-{"question":"Who is the best tennis player of all time ?", "ground_truth":"Roger Federer"}
\ No newline at end of file
+{"query":"When was United Stated found ?", "response":"1776", "question":"When was United Stated found ?"}
```
Comment (singankit) [inline scenarios/evaluate-app-endpoint/data.jsonl:1]: Do we need question for any evaluator anymore ?

**Review by singankit — APPROVED** [review]

File: scenarios/evaluate-app-endpoint/data.jsonl (hunk 1 above)
Comment (minthigpen) [reply scenarios/evaluate-app-endpoint/data.jsonl:1]: No

Comment (w-javed) [reply scenarios/evaluate-app-endpoint/data.jsonl:1]: Let's remove question and use query only.

File: scenarios/evaluate-model-endpoints/evaluate-models-target.ipynb [hunk 2]
```diff
@@ -240,16 +252,16 @@
     "            \"similarity\": similarity_evaluator,\n",
     "        },\n",
     "        evaluator_config={\n",
-    "            \"content_safety\": {\"question\": \"${data.question}\", \"answer\": \"${target.answer}\"},\n",
-    "            \"coherence\": {\"answer\": \"${target.answer}\", \"question\": \"${data.question}\"},\n",
-    "            \"relevance\": {\"answer\": \"${target.answer}\", \"context\": \"${data.context}\", \"question\": \"${data.question}\"},\n",
+    "            \"content_safety\": {\"query\": \"${data.question}\", \"response\": \"${target.response}\"},\n",
+    "            \"coherence\": {\"response\": \"${target.response}\", \"query\": \"${data.question}\"},\n",
+    "            \"relevance\": {\"response\": \"${target.query}\", \"context\": \"${data.context}\", \"query\": \"${data.question}\"},\n",
```
Comment (w-javed) [inline scenarios/evaluate-model-endpoints/evaluate-models-target.ipynb:257]: should be {target.respopnse}

**Review by luigiw — APPROVED** [review]

**Pull request merged**